File: app/src/main/java/com/example/mutidemo/util/callback/ILocationListener.java
package com.example.mutidemo.util.callback;

import android.location.Location;

import com.amap.api.location.AMapLocation;

public interface ILocationListener {
    void onLocationGet(Location location);//原生GPS定位数据

    void onAMapLocationGet(AMapLocation aMapLocation);//高德定位数据
}

File: app/src/main/java/com/example/mutidemo/callback/ILocationListener.kt
package com.example.mutidemo.callback

import android.location.Location
import com.amap.api.location.AMapLocation

interface ILocationListener {
    fun onLocationGet(location: Location?) //原生GPS定位数据
    fun onAMapLocationGet(aMapLocation: AMapLocation?) //高德定位数据
}